frontend/packages/jobs/src/functions/processSavePullRequest.ts

```diff
@@ -11,7 +11,7 @@ export type SavePullRequestResult = {
   success: boolean
   prId: number
   repositoryId: number
-  schemaFiles: Array<{
+  schemaFile: Array<{
```

Comment:
still array. but it might be no problem

Reply:
That's right. There's something wrong here, so I'll fix it!